code/game/machinery/traitorous_cloner.dm

```diff
@@ -0,0 +1,360 @@
+//Experimental cloner; clones a body regardless of the owner's status, letting a ghost control it instead
+
+/obj/machinery/clonepod/traitorous
+	name = "experimental cloning pod traitorous"
+	desc = "An ancient cloning pod. It seems to be an early prototype of the experimental cloners used in Nanotrasen Stations."
+	icon = 'icons/obj/machines/cloning.dmi'
+	icon_state = "pod_0"
+	req_access = null
+	circuit = /obj/item/circuitboard/machine/clonepod/traitorous
+	internal_radio = FALSE
+	var/emmaged = FALSE
```

Comment:
```suggestion
	var/emagged = FALSE
```

Reply:
Why make a suggestion with renaming variable without changing it everywhere so it won't break anything.

Reply:
Well, just pointing it out